### PR TITLE
OCM-10998 | test: fix id:57094

### PR DESCRIPTION
**Pull request by jameszwang**

### CODE REVIEW

File: tests/e2e/test_rosacli_upgrade.go [hunk 1]
```diff
@@ -752,7 +752,7 @@ var _ = Describe("Describe/List rosa upgrade",
 					By("Check list upgrade")
 					out, err := upgradeService.ListUpgrades(clusterID)
 					Expect(err).To(BeNil())
-					Expect(out.String()).To(MatchRegexp(`%s\s+recommended\s+-\s+scheduled for %s %s UTC`,
+					Expect(out.String()).To(MatchRegexp(`%s\s+recommended\s+-\s+(scheduled|started) for %s %s UTC`,
```
[the remark this answers is not in the document]
Comment (jameszwang) [reply tests/e2e/test_rosacli_upgrade.go:755]: If the upgrade scheduled by other case, the schedule time is diffrent.However, if it is scheduled by this case itself, the scheduled/started time will be consistent with the one in the assertion.

[the remark this answers is not in the document]
Comment (AkashKanni) [reply tests/e2e/test_rosacli_upgrade.go:755]: okay, Thanks Zhe for the information.

**Pull request merged**